### PR TITLE
chore!: Normalize repository, dropping node <10.13 support

### PR DESCRIPTION
**Pull request by sttk**
This PR updates this project:

* Updates dependency versions
* Introduces GitHub Actions
* Adds, changes and deletes some project files

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015, 2017 Blaine Bublitz <blaine.bublitz@gmail.com>, Eric Schoffstall <yo@contra.io> and other contributors
+Copyright (c) 2015, 2017, 2021 Blaine Bublitz <blaine.bublitz@gmail.com>, Eric Schoffstall <yo@contra.io> and other contributors
```
Comment (phated) [inline LICENSE]:
```suggestion
Copyright (c) 2015, 2017, 2021 Blaine Bublitz <blaine.bublitz@gmail.com> and Eric Schoffstall <yo@contra.io>
```

File: README.md [hunk 2]
```diff
@@ -29,20 +29,19 @@ Takes a version and array of ranges, returns the greatest satisfied range. Range
...
 
 [coveralls-url]: https://coveralls.io/r/gulpjs/semver-greatest-satisfied-range
-[coveralls-image]: http://img.shields.io/coveralls/gulpjs/semver-greatest-satisfied-range/master.svg
+[coveralls-image]: https://img.shields.io/coveralls/gulpjs/semver-greatest-satisfied-range/master.svg?style=flat-square
+<!-- prettier-ignore-end -->
 
-[gitter-url]: https://gitter.im/gulpjs/gulp
-[gitter-image]: https://badges.gitter.im/gulpjs/gulp.svg
+<!-- prettier-ignore-start -->
+[range-support]: https://github.com/phated/sver-compat#range-support
```
Comment (phated) [inline README.md]: This needs to be updated to the official `sver` module. The documentation about "sver-compat" need to be updated to reference the "sver" project.

File: package.json [hunk 3]
```diff
@@ -9,32 +9,37 @@
...
-    "lint": "eslint . && jscs index.js test/",
+    "lint": "eslint .",
     "pretest": "npm run lint",
-    "test": "mocha --async-only",
-    "cover": "istanbul cover _mocha --report lcovonly",
-    "coveralls": "npm run cover && istanbul-coveralls"
+    "test": "nyc mocha --async-only"
   },
   "dependencies": {
     "sver-compat": "^1.5.0"
```
Comment (phated) [inline package.json]: This needs to use `sver` instead of my fork.

**Review by phated — CHANGES_REQUESTED** [review]
Just needs to be updated to use the main `sver` package instead of my fork with node 0.10 support.

**Comment by sttk** [comment]
@phated I've modified what you pointed out.

I replaced `sver-compat` to `sver` but a test about prerelease version caused an error. Therefore I changed the 3rd parameter of `SemverRange.match` from `true` to `false`, then the test was passed. I don't know whether this modification is correct.

**Comment by phated** [comment]
Thank you for identifying that change and telling me. I've thought a lot about this and tested some things locally and I feel that our tests were testing our desired behavior—that a prerelease version is not covered by the final release, since there could be breaking changes.

However, there were bugs in sver-compat that weren't handling this correctly which I found while writing some tests against it. For example, `1.0.1-alpha.1` should also not be satisfied by `^1.0.0` because there might be changes that require a patch release in other packages to be compatible. I've added that test on this branch.

**Review by phated — APPROVED** [review]
Thanks for all your work on this! 🙇

**Pull request merged**